src/pykx/pandas_api/pandas_meta.py

```diff
@@ -154,6 +154,31 @@ def mean(self, axis: int = 0, numeric_only: bool = False):
             tab
         )
 
+    @api_return
+    def std(self, axis: int = 0, ddof: int = 1, numeric_only: bool = False):
+        tab = self
+        if 'Keyed' in str(type(tab)):
+            tab = q('{(keys x) _ 0!x}', tab)
```

Comment:
Safer to use `value tab` in this instance, you'll run into an issue with the current method if the keyed side of the table has duplicate named columns to the values part (I ran into this in testing)

```
q)tab:([10?1f;10?1f]10?1f;10?1f)
q)tab
x          x1       | x          x1       
--------------------| --------------------
0.391543   0.4707883| 0.4931835  0.3927524
0.08123546 0.6346716| 0.5785203  0.5170911
0.9367503  0.9672398| 0.08388858 0.5159796
0.2782122  0.2306385| 0.1959907  0.4066642
0.2392341  0.949975 | 0.375638   0.1780839
0.1508133  0.439081 | 0.6137452  0.3017723
0.1567317  0.5759051| 0.5294808  0.785033 
0.9785     0.5919004| 0.6916099  0.5347096
0.7043314  0.8481567| 0.2296615  0.7111716
0.9441671  0.389056 | 0.6919531  0.411597 
q){keys[x] _ 0!x}tab
'rank
```

Additionally it's more performant (albeit neither are particularly inefficient)

```
q)\t:10000 {keys[x] _ 0!x}tab
21
q)\t:10000 value tab
1
```

Reply:
Hi Conor.
I have applied your suggestion, but I'm still left with a couple of questions:

- Doesn't this apply to other functions like `mean`? I can also see that it's used in `preparse_computation` . We discussed the use of `value` internally but ultimately decided to go with `{keys[x] _ 0!x}` since it was used all throughout the `pandas_meta` file.
- Would it be better to use `q.value(tab)` or `q('value', tab)`? We lean towards the first option because it seems more clean, but we can't really tell if the use of the library itself inside itself is permitted / recommended.

Reply:
Yes it would apply for those examples/versions too I'll look to have the team update the general case! I think the first option is probably more sensible as it's cleaner like you say

Reply:
Okay. I have changed the function call to `q.value(tab)`